### PR TITLE
Add button to new GH repo overview

### PR DESCRIPTION
**Pull request by svenefftinge**
- Adds the button to the new repo overview
- updates webpack and other libs 
- adds source maps to the build for better debuggability
- makes gp button on issues secondary.
- moves gp button on PRs to the right

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,19 +1,28 @@
-'use strict';
 const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 module.exports = {
-    mode: 'production',
```
Comment (Siddhant-K-code) [inline webpack.config.js:5]: **Suggestion (non-blocking)** We should keep this. Or if we really want the dev. mode we can pass `development`. To patch this Build warning -- 

image

**Review by Siddhant-K-code — APPROVED** [review]
This works like a charm 😍❤️ 

- ✨  This even works with the latest layout (in Private Beta) of GitHub:
  image

- Code LGTM (have some minor non-blocking comments) ✅ 

---

One thing, we should do is bump [browser extension version in the manifest file](https://github.com/gitpod-io/browser-extension/blob/master/manifest.json#L5) to `1.20`.

https://github.com/gitpod-io/browser-extension/blob/748887b158e2c850f9d0f8da3cbbbda33ceaf408/manifest.json#L5

This is because, when we would release the extension to Browser Extension stores, it would trigger browser APIs to update the extension to latest.

**Review by filiptronicek — APPROVED** [review]
LGTM ✅

**Comment by gtsiolis** [comment]
Closing also https://github.com/gitpod-io/gitpod/issues/14117 and https://github.com/gitpod-io/gitpod/issues/18375.

**Pull request merged**